packages/server/graphql/public/typeDefs/FeatureFlag.graphql

```diff
@@ -18,4 +23,9 @@ type FeatureFlag {
   The scope of the feature flag
   """
   scope: FeatureFlagScope!
+
+  """
+  Whether the flag is enabled for an owner or not
```

Comment:
-1 `enabled` is a property of FeatureFlagOwner, so a type like `UpdateFeatureFlagSuccess` that returns a FeatureFlag won't have enabled as a field. This is technically OK since `enabled` is optional, but it's not very expressive because we're basically using the same `FeatureFlag` type for 2 different types of objects (a FeatureFlag, and a FeatureFlagWithOwner). i would keep `enabled` off this object or make a second object like `OwnedFeatureFlag` to express that they're different